### PR TITLE
Fix various grammatical and formatting errors

### PR DESCRIPTION
**Pull request by bsjcho**
Just suggestions - please modify/remove changes as necessary.

### CODE REVIEW

File: docs/gameplay-multiplayer-server-multiplayer.md [hunk 1]
```diff
@@ -1004,7 +1004,7 @@ _Parameters_
 | dispatcher | [Dispatcher](#match-runtime-api-go) exposes useful functions to the match, and may be used by the server to send messages to the participants of the match. |
 | tick | Tick is the current match tick number, starts at 0 and increments after every `MatchLoop` call. Does not increment with calls to `MatchJoinAttempt`, `MatchJoin`, or `MatchLeave`. |
 | state | Custom match state interface, use this to manage the state of your game. You may choose any structure for this interface depending on your game's needs. |
-| graceSeconds | Messages is a list of data messages received from users between the previous and current tick. |
+| graceSeconds | The number of seconds provided to complete a graceful termination before a match is forcefully closed. |
```
Comment (bsjcho) [inline docs/gameplay-multiplayer-server-multiplayer.md:1007]: Please review/replace - I had a quick go at the parameter definition here.

File: docs/runtime-code-basics.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ By default, the server will scan all files within the "data/modules" folder rela
 nakama --runtime.path "$HOME/some/path/"
 ```
 
-All files with the ".lua" or ".so" extensions found in the runtime path will be loaded and evaluated as part of the boot up sequence. Each Lua file represents a module and all code in each module will be run and can be used to register functions which can operate on messages from clients as well as execute logic on demand. Shared Object files are the equivalent for Go plugins.
+All files with the ".lua" or ".go" extensions found in the runtime path will be loaded and evaluated as part of the boot up sequence. Each Lua file represents a module and all code in each module will be run and can be used to register functions which can operate on messages from clients as well as execute logic on demand. Shared Object files are the equivalent for Go plugins.
```
Comment (bsjcho) [inline docs/runtime-code-basics.md:20]: Unsure if this was a typo or not given the below reference to `Shared Object` files, which seemingly have the `.so` extension.

File: docs/gameplay-multiplayer-server-multiplayer.md (hunk 1 above)
Comment (novabyte) [reply docs/gameplay-multiplayer-server-multiplayer.md:1007]: Thanks 🙏

File: docs/runtime-code-basics.md (hunk 2 above)
Comment (novabyte) [reply docs/runtime-code-basics.md:20]: This is not a typo. The server loads the compiled Go code (a shared object compiled with the `-buildmode=plugin` option in the Go compiler). I'll revert it back after this content is merged.

**Review by novabyte — APPROVED** [review]
Nice changes. Thanks @bsjcho 👍

**Pull request merged**